packages/compartment-mapper/src/types.js

```diff
@@ -1,7 +1,7 @@
 // @ts-check
 /// <reference types="ses"/>
 
-export const moduleJSDocTypes = true;
+export {};
```

Comment:
What's this?

Reply:
TypeScript treats a JavaScript file that contains only JSDoc comments with type annotations as a source of “ambient types”, but we don’t want that, so we need one executable line of JavaScript. This is just an update for. the simpler idiom we’ve converged on at Agoric, effectively a noöp.